docs/contributing.rst

```diff
@@ -32,7 +32,7 @@ the team and contributors are working on, you can join our public team calls:
 
 - Mondays and Wednesdays at 3pm CET/CEST.
 
-Both calls take place on `Jitsi <https://meet.ethereum.org/solidity>`_.
+Both calls take place on `Jitsi <http://meet.soliditylang.org/>`_.
```

Comment:
Shouldn't the URL be HTTPS instead of HTTP?

Reply:
yeah the cert isn't working yet, but devops is on it

Reply:
cert is working now!

Reply:
"Expires: Monday, 31 October 2022" I'm assuming we are auto-renewing them.